sources/packages/api/src/cra-integration/cra-personal-verification.service.ts

```diff
@@ -72,7 +78,10 @@ export class CRAPersonalVerificationService {
     const craRecords = students.map((student) => {
       return this.createCRARecordFromStudent(
         student,
-        STUDENT_SIN_VALIDATION_TAG,
+        this.createFreeProjectArea(
+          STUDENT_SIN_VALIDATION_TAG,
```

Comment:
We need to reduce the size of this constant. Before it was used just as a "label" but now we are adding the record id also. If I am not wrong the total size of the _free project area_ is 30, right?
Can we rename and change the value of this const as below:
```typescript
const SIN_VALIDATION_TAG = "SIN_VALIDATION";
```